### DISCUSSION TITLE
(PLEASE HELP) Video Capture function produces cv::cvtcolor error

### DISCUSSION THREAD

**Discussion by yumeISyou** [datasciencecompleteguide/Machine-Learning#32] [Q&A] [answered]
I saw your video about this issue and I already set the VideoCapture is (0), still have error. (QAQ) 
Can you hlep me...

All code:

![image](https://user-images.githubusercontent.com/97496169/148881683-4b7b1618-6d03-486f-ae98-b2aa901bb2e4.png)

![image](https://user-images.githubusercontent.com/97496169/148881694-e31853bd-9193-4337-bcb9-3a68cf4dec96.png)

![image](https://user-images.githubusercontent.com/97496169/148882440-8780911b-8f1d-4f65-b617-833cb0dc9a58.png)

![image](https://user-images.githubusercontent.com/97496169/148882453-1d664cfd-14de-4d52-b54f-50b72d5ac666.png)

The error
![image](https://user-images.githubusercontent.com/97496169/148881924-12bb7374-ec3c-4303-81f8-1ce785607428.png)

**Comment by datasciencecompleteguide** [comment, marked as answer]
Hiii,
Please check one more video and I hope this will resolve your query.
And in case if it didn't get solved than revert me back here only, will try with some other solution.
https://youtu.be/hwUQ2_QAdsA

**Reply by yumeISyou** [reply]
yes, I follow this video to type the code
and the result is :

![image](https://user-images.githubusercontent.com/97496169/148900123-f8b14d73-336c-4e3c-b9cc-70b71c04d174.png)

![image](https://user-images.githubusercontent.com/97496169/148900155-a24ba534-ad62-465d-8c51-2ee9c38d264c.png)

and In my origin code already is (0), so I think it doesn't work (QAQ)

**Reply by yumeISyou** [reply]
I solve it!! I change to another pc to run, it is actually can run, thank you~

**Reply by datasciencecompleteguide** [reply]
That's Great....You yourself sorted it out!! Appreciable.👍👍
